src/app/containers/app/app.component.ts

```diff
@@ -1,10 +1,24 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import * as fromSingleFeeAccountStore from '../../../fee-accounts/store';
+import { Observable, of } from 'rxjs';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'app';
+
+  identityBar: Observable<any>;
+
+  constructor(
+    private store: Store<fromSingleFeeAccountStore.FeeAccountsState>
+  ) { }
+
+  ngOnInit() {
+
+    this.identityBar = of(this.store.pipe(select(fromSingleFeeAccountStore.getSingleFeeAccountArray)));
```

Comment:
something is definitely wrong here... lets look at this together....

Reply:
This is also done